src/CoreWCF.NetTcp/tests/TaskBasedOperationTests.cs

```diff
@@ -86,11 +88,12 @@ public async Task StreamedSynchronouslyCompletingTask()
                 System.ServiceModel.ChannelFactory<Contract.ITaskService> factory = null;
                 Contract.ITaskService channel = null;
                 host.Start();
+
                 try
```

Comment:
Nit: extra blank line introduced

Reply:
Fixed. I think there is another PR for an `.EDITORCONFIG` file that would allow us to lint to your preferences, or you could create your own for that purpose.